CONTRIBUTING.md

```diff
@@ -39,9 +39,38 @@ make docker-build
 It is possible to specify a different image name and tag.
 
 ```shell
-make docker-build IMG=exmaple.com/spegel TAG=feature
+make docker-build IMG=example.com/spegel:feature
```

Comment:
Leave this as it is and lets fix the Makefile in a separate PR.

Reply:
Done.